crates/core/src/transport/udp/udp_transport.rs

```diff
@@ -0,0 +1,45 @@
+use super::*;
+use crate::transport::udp::udp_connection::UdpConnection;
+
+struct UdpTransport {
+    connections: DashMap<(IpAddr, u16), UdpConnection>,
+    channel: (
+        mpsc::Sender<InternalMessage>,
```

Comment:
what are this channels for?

Reply:
My idea was that inbound packets which didn't have an associated connection would be sent to one of these channels, and a separate spawned process would read them - but I'm still figuring it out.